docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,217 @@
+data.head(2)
+
+
+
+#%%
+# Split Data
+# -------------
+
+# convert date column to datetime, `issue_d`` is date column
+data['issue_d'] = pd.to_datetime(data['issue_d'])
+
+# Use data from June and July for train and August for test:
+train_df = data[data['issue_d'].dt.month.isin([6, 7])]
+test_df = data[data['issue_d'].dt.month.isin([8])]
+
+
+#%%
+# Run Deepchecks for Train Test Validation
+# ===========================================
+#
+# Define a Dataset Object
+# -------------------------
+#
+# Create a deepchecks Dataset, including the relevant metadata (label, date, index, etc.).
+# Check out :class:`deepchecks.tabular.Dataset` to see all of the columns that can be declared.
+
+
+#%%
+# Define Lending Club Metadata
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+categorical_features = ['addr_state', 'application_type', 'home_ownership', \
+  'initial_list_status', 'purpose', 'term', 'verification_status', 'sub_grade']
+index_name = 'id'
+label = 'loan_status' # 0 is DEFAULT, 1 is OK
+datetime_name = 'issue_d'
+
+
+#%%
+# Create Dataset
+# ^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+from deepchecks.tabular import Dataset
+
+# We explicitly state the categorical features,
+# otherwise they will be automatically inferred,
+# which may not work perfectly and is not recommended.
+
+# The label can be passed as a column name or
+# as a separate pd.Series / pd.DataFrame
+
+# all metadata attributes are optional.
+# Some checks require specific attributes and otherwise will not run.
+
+train_ds = Dataset(train_df, label=label,cat_features=categorical_features, \
+                   index_name=index_name, datetime_name=datetime_name)
+test_ds = Dataset(test_df, label=label,cat_features=categorical_features, \
+                   index_name=index_name, datetime_name=datetime_name)
+
+#%%
+
+# for convenience lets save it in a dictionary so we can reuse them for future Dataset initializations
+columns_metadata = {'cat_features' : categorical_features, 'index_name': index_name,
+                    'label':label, 'datetime_name':datetime_name}
+
+#%%
+# Run the Deepchecks Suite
+# --------------------------
+#
+# Validate your data with the :func:`deepchecks.tabular.suites.train_test_validation` suite.
+# It runs on two datasets, so you can use it to compare any two batches of data (e.g. train data, test data, a new batch of data
+# that recently arrived)
+#
+# Check out the :doc:`when should you use </getting-started/when_should_you_use>`
+# deepchecks guide for some more info about the existing suites and when to use them.
+
+from deepchecks.tabular.suites import train_test_validation
+
+validation_suite = train_test_validation()
+suite_result = validation_suite.run(train_ds, test_ds)
+# Note: the result can be saved as html using suite_result.save_as_html()
+# or exported to json using suite_result.to_json()
+suite_result
+
+#%%
+# We can see that we have a problem in the way we've split our data!
```

Comment:
Would explain what you're referring to. "As you can see in the conditions table at the beginning of the suite output, our data failed on the check Date Train-Test Leakage, which checks for ...". Otherwise it's unclear how you got to this conclusion.